openedx/core/djangoapps/schedules/templates/schedules/edx_ace/upgradereminder/email/body.html

```diff
@@ -33,7 +33,7 @@
             <a href="{{ upsell_link }}">
                 <img
                     src="{{ cert_image }}"
-                    alt="{% blocktrans %}Example print-out of a verified certificate{% endblocktrans %}"
+                    alt="{% trans 'Example print-out of a verified certificate' %}"
```

Comment:
why was this change needed?

Reply:
It's not needed, Clinton mentioned to me in a prior review that using trans is preferable to blocktrans  when possible.

Reply:
ok. gotcha.